iron-validatable-behavior.html

```diff
@@ -16,6 +16,40 @@
   /**
    * Use `Polymer.IronValidatableBehavior` to implement an element that validates user input.
    *
+   * This element can be used in two different ways:
+   *   1. to add a `validate()` method to a custom element that you are implementing,
+   * thus making it validatable
+   *   2. to add custom validation logic to an existing `<iron-input>`.
```

Comment:
Is it worth mentioning somewhere that there are approximately three kinds of validation, and this might not be the kind they're looking for? I'm making this all up here, but I'm thinking of:

- pattern validation, used to prevent inappropriate input _before_ it appears in the input box.
- field validation (the validate() method), which by default ins't invoked until the user tries to submit a form.
- proactive field validation, where an application validates form fields as the user types and only enables a submit button when all fields are valid.

Hmm... No, it's probably more appropriate to get into that in a separate article on form itself. But at the minimum, you might explain _when_ validate is called. For example, "By default, an `<iron-form>` element validates its fields when the user presses the `submit` button. To validate a form imperatively, call the form's `validate` method."